### PR TITLE
Update content and navigation for kickout

### PR DESCRIPTION
**Pull request by TheNickWilson**

### CODE REVIEW

File: conf/views/useDifferentService.njk [hunk 1]
```diff
@@ -16,6 +16,15 @@
           {{ messages("useDifferentService.heading") }}
         </h1>
 
+        <p class="govuk-body">
+          {{ messages("useDifferentService.guidance.1") }}
+          <a href="https://www.gov.uk/new-computerised-transit-system" class="govuk-link">{{ messages("useDifferentService.link") }}</a>
```
Comment (garethgooch) [inline conf/views/useDifferentService.njk]: Should we be hard coding the urls in the views? is this not something we can grab from appConf?

File: app/navigation/Navigator.scala [hunk 2]
```diff
@@ -53,7 +53,7 @@ class Navigator @Inject()() {
     if (ua.get(GoodsLocationPage).contains(GoodsLocation.BorderForceOffice)) {
       routes.PresentationOfficeController.onPageLoad(ua.id, NormalMode)
     } else {
-      routes.AuthorisedLocationController.onPageLoad(ua.id, NormalMode)
+      routes.UseDifferentServiceController.onPageLoad(ua.id)
```
Comment (garethgooch) [inline app/navigation/Navigator.scala]: Should we be passing the mrn here? we appear to pass this from the controller to the view, but its then not used?

File: conf/views/useDifferentService.njk (hunk 1 above)
Comment (TheNickWilson) [reply conf/views/useDifferentService.njk]: This URL is static - i.e. it's the same in every environment, so having it in config doesn't offer any advantage over just having it in the view.

File: app/navigation/Navigator.scala (hunk 2 above)
Comment (TheNickWilson) [reply app/navigation/Navigator.scala]: The page is part of a journey, so has the MRN in the URL.  You're right that we currently don't use it on the view, but that will quite possibly change in future (e.g. we may want a way for the user to get back to change their answer, in which case we need the MRN).

**Pull request merged**